modelci/experimental/finetuner/trainer.py

```diff
@@ -130,20 +132,25 @@ def from_training_job(cls, training_job: TrainingJob) -> 'PyTorchTrainer':
                 'num_sanity_val_steps': 0,
                 'gpus': 1,  # TODO: set GPU number
                 **trainer_kwargs,
-            }
+            },
+            model_id=training_job.model
         )
         return trainer
 
     def start(self):
         def training_done_callback(future):
             model_train_curd.update(TrainingJobUpdate(_id=self._id, status=Status.PASS))
-            # TODO: save to database
+            # TODO: save to database and update model_status
             print(self.export_model())
 
         self._task = self._executor.submit(self.trainer_engine.fit, self.model, **self._data_loader_kwargs)
         self._task.add_done_callback(training_done_callback)
         model_train_curd.update(TrainingJobUpdate(_id=self._id, status=Status.RUNNING))
 
+        model_bo = ModelService.get_model_by_id(self.model_id)
+        model_bo.model_status.append(ModelStatus.TRAINING)
```

Comment:
when the model is training, you should remove the `draft` tag, and add a `training` tag